lua/config/lualine.lua

```diff
@@ -4,13 +4,14 @@ require("lualine").setup({
     options = {
         icons_enabled = true,
         theme = "auto",
-        -- component_separators = { left = "", right = "" },
-        -- section_separators = { left = "", right = "" },
+        component_separators = { left = "", right = "" },
+        section_separators = { left = "", right = "" },
         disabled_filetypes = {},
         always_divide_middle = true,
+        globalstatus = false,
     },
     sections = {
-        lualine_a = {},
+        lualine_a = {'mode'},
```

Comment:
Since the cursor changes from square to line when in insert mode. I don't find this useful.